scrape_bonds.py

```
@@ -0,0 +1,298 @@
+DEFAULT_OUTDIR = "output"
+STORAGE_STATE = "tv_storage_state.json"
+WAIT_TIMEOUT_MS = 30000  # 30s
+
+def norm_pct(txt: str) -> Optional[float]:
+    if not txt:
+        return None
+    t = txt.strip().replace('%', '').replace(',', '')
+    # Some cells like '102.76%'
+    try:
+        return float(t)
+    except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)